lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+module Spaceship
+  module Tunes
+    class AppAnalytics < TunesBase
+      # @return (Integer) The app ID to grab analytics for
+      attr_accessor :apple_id
+
+      attr_mapping({
+        'adamId' => :apple_id
+      })
+
+      class << self
+        def factory(attrs)
+          return self.new(attrs)
+        end
+      end
+
+      def app_units
+        start_t, end_t = time_last_7_days
+
+        app_units_interval(start_t, end_t)
+      end
+
+      def app_views
+        start_t, end_t = time_last_7_days
+
+        app_views_interval(start_t, end_t)
+      end
+
+      def app_in_app_purchases
+        start_t, end_t = time_last_7_days
+
+        app_in_app_purchases_interval(start_t, end_t)
+      end
+
+      def app_sales
+        start_t, end_t = time_last_7_days
+
+        app_sales_interval(start_t, end_t)
+      end
+
+      def app_paying_users
+        start_t, end_t = time_last_7_days
+
+        app_paying_users_interval(start_t, end_t)
+      end
+
+      def app_installs
+        start_t, end_t = time_last_7_days
+
+        app_installs_interval(start_t, end_t)
+      end
+
+      def app_sessions
+        start_t, end_t = time_last_7_days
+
+        app_sessions_interval(start_t, end_t)
+      end
+
+      def app_active_devices
+        start_t, end_t = time_last_7_days
+
+        app_active_devices_interval(start_t, end_t)
+      end
+
+      def app_active_last_30_days
+        start_t, end_t = time_last_7_days
+
+        app_active_last_30_days_interval(start_t, end_t)
+      end
+
+      def app_crashes
+        start_t, end_t = time_last_7_days
+
+        app_crashes_interval(start_t, end_t)
+      end
+
```

Comment:
I'd want to keep the interval methods exposed if only the give the option to the client to determine their own time interval. The request takes any interval, maybe adding an interval formating function would make the use of this more convenient.